lib/FullScreenContainer.js

```diff
@@ -245,7 +264,21 @@ export default class FullScreenContainer extends React.Component {
   }
 
   _renderScrollableContent() {
-    const { dataSource, mediaList } = this.props;
+    const { dataSource, mediaList, useGallery } = this.props;
+    if (useGallery) {
+      return (
+        <Gallery
```

Comment:
I'v used Gallery.js forked from ldn0x7dc/react-native-gallery which support costuming item for renderPage.